packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+/// Default localized resource values for the material widgets.
+///
+/// This class is just a placeholder, it only provides english values.
+class MaterialLocalizations {
+  const MaterialLocalizations._(this.locale) : assert(locale != null);
+
+  /// The locale that the values of this class's localized resources
+  /// have been translated for.
+  final Locale locale;
+
+  /// Creates an object that provides default localized resource values for the
+  /// material widgets.
+  ///
+  /// This method is typically used to create a [LocalizationsDelegate].
+  /// The [MaterialApp] does so by default.
+  static Future<MaterialLocalizations> load(Locale locale) {
+    return new SynchronousFuture<MaterialLocalizations>(new MaterialLocalizations._(locale));
+  }
+
+  /// The `MaterialLocalizations` from the closest [Localizations] instance
+  /// that encloses the given context.
+  ///
+  /// This method is just a convenient shorthand for:
+  /// `Localizations.of<MaterialLocalizations>(context, MaterialLocalizations)`.
+  ///
+  /// References to the localized resources defined by this class are typically
+  /// written in terms of this method. For example:
+  /// ```dart
+  /// tooltip: MaterialLocalizations.of(context).backButtonTooltip,
+  /// ```
+  static MaterialLocalizations of(BuildContext context) {
+    return Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
+  }
+
+  /// The tooltip for the leading [AppBar] menu (aka 'hamburger') button
+  String get openAppDrawerTooltip => 'Open navigation menu';
+
+  /// The [BackButton]'s tooltip.
+  String get backButtonTooltip => 'Back';
+
+  /// The [CloseButton]'s tooltip.
+  String get closeButtonTooltip => 'Close';
+
+  /// The tooltip for the [MonthPicker]'s next month button.
```

Comment:
"next month" should be in quotes otherwise it sounds like you're talking about the subsequent "month" button from a list of "month" buttons